vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -225,39 +229,39 @@ An overview of the current connection and profile information is provided via `e
 ```
 ent pro new [profileName] [EntandoAppName] [namespace]
 
-This sets minimal profile data and explains next steps.
+This sets the minimal profile data and explains next steps.
 ```
 
 2. Link the current profile to a Kubernetes context (alias of `ent attach-kubectx`)
 ```
 ent pro link [contextName]
 ```
-This instructs `ent` how to connect to the Kube containing the EntandoApp.
+This instructs ent how to connect to the Kubernetes containing the Entando Application.
 
-3. Activate the profile that `ent` should use across shells
+3. Activate the profile that ent should use across shells
 ```
 ent pro use [profileName]
 ```
 
-This attempts to fully restore a profile type by considering the login and related Entando applications.
+This attempts to fully restore a profile type by considering the login and related Entando Applications.
 
-4. Activate the profile that `ent` should use within the current shell
+4. Activate the profile that ent should use within the current shell
 ```
 source ent pro use [profileName]
 ```
 
-This allows `ent` instances in different shells to simultaneously use different Kube contexts, kubeconfigs or custom commands. The quickstart script creates a profile named "qs--{vmname}" that is associated with the Entando application it generates. 
+This allows ent instances in different shells to simultaneously use different Kube contexts, kubeconfigs or custom commands. The quickstart script creates a profile named "qs--{vmname}" that is associated with the Entando Application it generates. 
 
-4. Print a list of the available profiles
+5. Print a list of the available profiles
 ```
 ent pro list
 ```
 
-5. Delete a profile
+6. Delete a profile
 ```
 ent pro delete [profileName]
 ```
-This also deletes the VM of an EntandoApp directly managed by `ent`.
+This also deletes the VM of an Entando Application directly managed by ent.
```

Comment:
@w-caffiero-entando is this statement true? It's surprising behavior that deleting a profile would also delete a VM. I guess I rarely delete a profile...

Reply:
No that's not true: the command only deletes the profile.
This was probably meant to automatically delete VMs there were automatically created by ent, but at the end the idea was dropped.

Reply:
@nshaw for the same reasons I'd fix the language that uses the concept of "VMs directly managed by ent", they are not.
In this case just remove the entire "This also deletes the VM of an Entando Application directly managed by ent."